native_mate/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2014 Cheng Zhao.
+
+Permission is hereby granted, free of charge, to any person obtaining
+a copy of this software and associated documentation files (the
+"Software"), to deal in the Software without restriction, including
+without limitation the rights to use, copy, modify, merge, publish,
+distribute, sublicense, and/or sell copies of the Software, and to
+permit persons to whom the Software is furnished to do so, subject to
+the following conditions:
+
+The above copyright notice and this permission notice shall be
+included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```

Comment:
if this isn't different to Electron's license we should drop this file

Reply:
Both are MIT, but the copyright holder is different so we may want to keep it.

That's probably a question for @zcbenz to address before this gets merged :smiley:

Reply:
Sublicensing from MIT to ...MIT?

Anyway, I'm not taking issue with licensing but rather that we need to give proper acknowledgment to the copyright holder, which brings us back to @zcbenz

Reply:
Hereby I grant Electron team the permission to replace `Copyright (c) 2014 Cheng Zhao` with `Copyright (c) 2013-2018 GitHub Inc.` in the license file.